### ISSUE TITLE
SOLID failed in extremely turbid waters

### ISSUE THREAD

**Issue by SidneyCCCC** [BrandonSmithJ/MDN#21]
![image](https://github.com/user-attachments/assets/e8405afe-9eeb-42fa-87ad-2ca1582d7b6f)
![image](https://github.com/user-attachments/assets/4c35d08e-85e3-4fac-9297-8dda8ccb1af1)
When I used SOLID on an extremely turbid waters (The Yangtze River Estuary), I found that it failed. Is it because there are no such turbid samples in the training set, or did you set a threshold?

**Comment by BrandonSmithJ** [comment]
No, there isn't any threshold like that applied. It's likely something to do with the Rrs being provided - can you share the Rrs values for one of the failed pixels?

**Comment by SidneyCCCC** [comment]
![image](https://github.com/user-attachments/assets/432f67a5-e321-420d-9eac-80d0d8f5d601)
This spectrum was generated with Acolite-DSF default settings. I don't know where it went wrong, but it turns out to be NAN in TSS.

**Comment by SidneyCCCC** [comment]
I would like to add that I have converted the result to Rrs before input it into SOLID.

**Comment by SidneyCCCC** [comment]
Sorry, I found that I selected the wrong model. SOLID is valid for this area. It is invalid in the model that is exported by tss, chl, and cdom at the same time. Why is this?

**Comment by BrandonSmithJ** [comment]
Can you show me what your code that generates the estimates looks like? In particular, the code that uses the `image_estimates` function? It would also help to see the actual Rrs values rather than the rhos values - it's quite possible acolite is generating invalid Rrs spectra, even if the rhos appears valid.
